content/nginx-one/nginx-configs/config-templates/add-multiple-services.md

```diff
@@ -0,0 +1,303 @@
+---
+nd-docs: null
+title: Add service-specific locations
+toc: true
+weight: 400
+type:
+- how-to
+---
+
+# Overview
+
+This guide shows how to extend working submission from the [Submit Templates Guide]({{< ref "submit-templates.md" >}}) by adding server augments for new services with dedicated location augments.
+
+**Note: With current limitation to not being able to retrieve previous submissions, you must provide entire request including any updates.**
+
+## Import template
+
+If there aren't existing augment templates that fit your needs, you will need to create and import them.
+
+### New server augment template
+
+Since we don't have a server augment yet, we need to create one. This template creates dedicated server blocks for each service.
```

Comment:
```suggestion
Because there isn’t a server augment yet, you need to create one. This template creates a dedicated server augment for each service.
```

Reply:
When referring to the user, use "you."